app/controllers/posts_controller.rb

```diff
@@ -9,6 +9,32 @@ def show
     @post = Post.find(params[:id])
   end
 
+  def new
+    @user = current_user
+    post = Post.new
+    respond_to do |format|
+      format.html { render :new, locals: { post: } }
+    end
+  end
```

Comment:
- [ ] I can't check the form ı am getting an error when clicking on the post or seeing the all post button. Please fix it.

![image](https://user-images.githubusercontent.com/38635158/173134191-ca4941d4-ec29-491e-9a93-a13a456f3c75.png)
_Next code reviewer please check if you can create post and comment in the page._

Reply:
Hi, I wasn't able to reproduce this error, I am able to open create post and comment forms.

Reply:
![image](https://user-images.githubusercontent.com/73714615/173173390-2f64a6b5-d390-4b44-bc9e-0b6ac6a3ea0d.png)
![image](https://user-images.githubusercontent.com/73714615/173173396-7a483e21-2ed6-450b-aa67-3b38993770a6.png)

Reply:
And about the number of posts, I implemented it so that the counters only get incremented when you create a post or comment, they are not calculated every time you visit the page. So if you create a post using the create post button (which I forgot to add in the last iteration - I have added it now) - then you can see the number of the posts incremented.

Reply:
Addressed